core/labels.go

```diff
@@ -36,10 +36,15 @@ var (
 		Key:         "pod_name",
 		Description: "The name of the pod",
 	}
+	// Deprecated label
 	LabelPodNamespace = LabelDescriptor{
 		Key:         "pod_namespace",
 		Description: "The namespace of the pod",
 	}
+	LabelNamespaceName = LabelDescriptor{
```

Comment:
Can we create a new version of storage schema instead of deprecating labels in the existing version?
It will be much easier for users to understand. WDYT?